README.adoc

```diff
@@ -323,11 +319,15 @@ application may take some time to get ready. Note: The latest version of most
 modern web browsers should have support for Server-Sent Events, with the
 exception of Internet Explorer.
 
+When you visit the URL, you should see something similar to the following webpage:
+
+image::webpage.png[Webpage screenshot, align="center"]
```

Comment:
I think including an image of the full webpage looks a bit jarring with the dark background of the webpage in the image on the light background of the text column. I think using an [`.adoc` table](https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#tables) will fit in more and might work better for users using screen readers.

Reply:
What would you think about cropping the image and only including the table portion: 

![image](https://user-images.githubusercontent.com/38739044/98299554-e0596b00-1f85-11eb-9d67-0088e5041e3a.png)

I think including a sample `.adoc` table lacks some value because it will neither have the same look as the table from the webpage, nor the same values, whereas a screenshot of the actual table is a better representation of a snapshot of the dynamic table they will see. 

The [rest-client-react-js](https://openliberty.io/guides//rest-client-reactjs.html#try-what-youll-build) guide seems to have done something similar.

What do you think?

Reply:
Seems fair, I think a cropped image can work too

Reply:
Kk I updated that.